pydoctor/driver.py

```diff
@@ -134,6 +134,10 @@ def getparser():
         '--buildtime', dest='buildtime',
         help=("Use the specified build time over the current time. "
               "Format: %s" % BUILDTIME_FORMAT))
+    parser.add_option(
+        '-W', action='store_true', dest='warnings_as_errors',
```

Comment:
For consistency, I think it would be good to have a long form of this option as well. Maybe just `--warnings-as-errors`.

Reply:
I will go for `--warnings-as-errors`